src/renderer/init.js

```diff
@@ -81,10 +81,14 @@ async function init() {
 
   const isMainWindow = remote.getCurrentWindow().name === "MainWindow";
 
-  if (initialSettings.hasPassword) {
+  let accounts;
+  try {
+    accounts = await getKey("app", "accounts", []);
+  } catch (e) {
+    // NB Failing to retrieve the accounts means the data is encrypted.
```

Comment:
That's a bit weird approach to me. What if it's just filesystem failure?. Can we at least filter with a specific error (if there is a way to identify it)

Reply:
🤔 you are right, needs a rethink